README.md

```diff
@@ -150,14 +150,26 @@ Activities require a certain formatting:
   id, // a string that uniquely represents this activity
   datetime, // a javascript Date object that represents the start of the activity
   label, // a string that represents the transaction
+  activityType: ACTIVITY_TYPE_PURCHASE,
   merchantDisplayName, // (optional) a string that represents the merchant where the purchase was made
-  purchaseCategory, // a string that represents the category of the purchase. Categories can be found here: https://github.com/tmrowco/northapp-contrib/blob/master/definitions.js
-  costAmount, // a floating point that represents the amount of the purchase
-  costCurrency, // a string that represents the currency in which the currency was made
+  purchaseCategory, // (optional) a string that represents the category of the purchase. Categories can be found here: https://github.com/tmrowco/northapp-contrib/blob/master/definitions.js
+  lineItems, // (required if the activity contains line items) an array with an object [{ identifier: xx, amount: 2.1, unit: 'kg', currency: XXX}] where `identifier` is a key from footprints.yml and `unit` a valid unit from definitions.js, and currency is only required if the line item is a monetary amount
```

Comment:
- This is always required for a purchase, I think? @corradio ?
- The example in comment is wrong - see suggestion below :)

```suggestion
  lineItems, // an array with specific items that can be either in monetary or amount-based form: [{ identifier: XX, value: 2.1, unit: XX}] where `identifier` is a key from footprints.yml and `unit` a valid unit from definitions.js. See purchase/index.test.js for examples.
```

Reply:
good catch!!